packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -410,106 +570,655 @@ class _CupertinoAlertActionSection extends StatelessWidget {
   // don't have many actions.
   final ScrollController scrollController;
 
-  bool get _layoutActionsVertically => children.length > 2;
+  @override
+  _CupertinoAlertActionSectionState createState() {
+    return new _CupertinoAlertActionSectionState();
+  }
+}
+
+class _CupertinoAlertActionSectionState extends State<_CupertinoAlertActionSection> {
+  static _CupertinoAlertActionSectionState of(BuildContext context) {
```

Comment:
you don't see to use this?

Reply:
Yep. Holdover from an InheritedWidget I had before. Removed.